docs/development.md

```diff
@@ -2,36 +2,37 @@
 
 This covers how to setup and configure a development environment using the Forklift tool suite.
 
- * [Development Environment Deployment](#development-environment-deployment)
-   * [Deploy a Katello Development Environment](#deploy-a-katello-development-environment)
-   * [Deploy a Stable Katello Development Box](#deploy-a-stable-katello-development-box)
-   * [Starting the Development Server](#starting-the-development-server)
-   * [Customizing the Development Environment](#customizing-the-development-environment)
-   * [Reviewing Pull Requests](#reviewing-pull-requests)
- * [Packit PR Builds](#packit-pr-builds)
- * [Test Puppet Module Pull Requests](#test-puppet-module)
- * [Hammer Development](#hammer-development)
- * [Capsule Development](#capsule-development)
- * [Client Development](#client-development)
- * [Dynflow](#dynflow-development)
- * [Smoker](#smoker)
- * [Advanced foreman features](#advanced-playbooks)
+* [Development Environment Deployment](#development-environment-deployment)
+    * [Deploy a Katello Development Environment](#deploy-a-katello-development-environment)
+    * [Deploy a Stable Katello Development Box](#deploy-a-stable-katello-development-box)
+    * [Starting the Development Server](#starting-the-development-server)
+    * [Customizing the Development Environment](#customizing-the-development-environment)
+    * [Reviewing Pull Requests](#reviewing-pull-requests)
+* [Packit PR Builds](#packit-pr-builds)
+* [Test Puppet Module Pull Requests](#test-puppet-module)
+* [Hammer Development](#hammer-development)
+* [Capsule Development](#capsule-development)
+* [Client Development](#client-development)
+* [Dynflow](#dynflow-development)
+* [Smoker](#smoker)
+* [Advanced foreman features](#advanced-features)
 
 ## Development Environment Deployment
 
 ### Deploy a Katello Development Environment
 
-A Katello development environment can be deployed on CentOS 7. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin. There are a variety of useful development environment options that should or can be set when creating a development box. These options are designed to configure your environment ready to use your own fork, and create pull requests. To create a development box:
+A Katello development environment can be deployed on Fedora and Red Hat flavor distributions. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin. There are a variety of useful development environment options that should or can be set when creating a development box. These options are designed to configure your environment ready to use your own fork, and create pull requests. To create a development box:
```

Comment:
I think we can use "Enterprise Linux" as the catch-all for any of the distros we support.

```suggestion
A Katello development environment can be deployed on Enterprise Linux (EL) distributions. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin. There are a variety of useful development environment options that should or can be set when creating a development box. These options are designed to configure your environment ready to use your own fork, and create pull requests. To create a development box:
```